tidb-lightning/tidb-lightning-physical-import-mode.md

```diff
@@ -11,23 +11,26 @@ Before you use the physical import mode, make sure to read [Requirements and res
 
 ## Implementation
 
-1. Before importing data, TiDB Lightning automatically switches the TiKV nodes to "import mode", which improves write performance and stops PD scheduling and auto-compaction.
+1. Before importing data, TiDB Lightning automatically switches the TiKV nodes to "import mode", which improves write performance and stops auto-compaction. TiDB Lightning determines whether to pause global scheduling according to the TiDB cluster version.
 
-2. `tidb-lightning` creates table schemas in the target database and fetches the metadata.
+    - When the TiDB cluster >= v6.1.0 and TiDB Lightning >= v6.2.0, TiDB Lightning pauses scheduling for the region that stores the target table data. After the import is completed, TiDB Lightning recovers scheduling.
+    - When the TiDB cluster < v6.1.0 or TiDB Lightning < v6.2.0, TiDB Lightning pauses global scheduling.
 
-3. Each table is divided into multiple contiguous **blocks**, so that Lightning can import data data from large tables (200 GB+) in parallel.
+2. TiDB Lightning creates table schemas in the target database and fetches the metadata.
 
-4. `tidb-lightning` prepares an "engine file" for each block to handle key-value pairs. `tidb-lightning` reads the SQL dump in parallel, converts the data source to key-value pairs in the same encoding as TiDB, sorts the key-value pairs and writes them to a local temporary storage file.
+3. Each table is divided into multiple contiguous **blocks**, so that TiDB Lightning can import data data from large tables (200 GB+) in parallel.
```

Comment:
```suggestion
3. Each table is divided into multiple contiguous **blocks**, so that TiDB Lightning can import data from large tables (200 GB+) in parallel.
```